src/template/related-search.ts

```diff
@@ -32,13 +32,15 @@ export function relatedResultsTemplate (contextualNavigation: { categories: any;
 
   if (contextualNavigation) {
     const { categories } = contextualNavigation
-    if (categories[0]?.name === 'topic') {
-      return html` <p class="related-search__title">Related search</p>
+    categories.array.forEach((element: any) => {
+      if (element?.name === 'topic') {
+        return html` <p class="related-search__title">Related search</p>
         <section class="related-search__tags">
             ${categories[0]?.clusters.map((item: any) =>
```

Comment:
Only thing I would suggest is consider replacing categories[0] with "element" here, because you're already in the loop.